ruby-homework/book.rb

```diff
@@ -0,0 +1,36 @@
+class Book
+    def initialize(title)
```

Comment:
You could initialize `title = ""` as an argument here and use `attr_accessor` instead of `attr_reader` so that the example in the homework works.
```
book = Book.new
book.title = "My Awesome Book"
book.add_chapter("My Awesome Chapter 1")
book.add_chapter("My Awesome Chapter 2")
book.chapters
# This should print:
# Your book: My Awesome Book has 2 chapters:
# 1. My Awesome Chapter 1
# 2. My Awesome Chapter 2
```

But I actually agree with you that your way is better :)